src/Utilities/DeprecationWarning.hh

```diff
@@ -0,0 +1,23 @@
+//---------------------------------Spheral++----------------------------------//
```

Comment:
This header needs to be added to the CMakeLists.txt file, otherwise the SpheralC stuff is missing the header.

Reply:
Good catch -- fixed